src/glomap/estimators/rotation_initializer.cc

```diff
@@ -4,124 +4,106 @@
 
 namespace glomap {
 
-bool ConvertRotationsFromImageToRig(
+bool InitializeRigRotationsFromImages(
```

Comment:
Some of this is similar to UpdateRigAndCameraCalibsFromReconstruction. Not too important to consolidate. Just FYI.